recipes/rainbowlog/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "rainbowlog" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rainbowlog-{{ version }}.tar.gz
+  sha256: 20eeda543dd6724622cc7a16f286a478b605e948ab263ca220da437cf6d73c5a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7,<4.0
```

```suggestion
    - python >=3.7
```
This can be removed, we will do this manually when the time comes.

@beckermr You had once recommended that, but I just wanted to double check if that's still true?